chess-AI/experimental-work/figshare_upload.py

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env python
```

Comment:
https://pastebin.com/RLV8m8vh

Reply:
This also shouldnt be on here, I will remove it.